engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableFaceCulling.java

```diff
@@ -19,7 +19,10 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Instances of this class enable OpenGL's Face Culling, i.e. to render only the inside faces of the skysphere.
+ * Enables OpenGL's Face Culling.
+ *
+ * This can be used to discard the fragments that face away from the camera, and can be used to improve
+ * the performance in certain situations like rendering only the inside faces of the skysphere.
```

Comment:
Hmmm. It's a good effort. The only problem is that it narrows the meaning of face culling a bit. I'd rewrite as:

```
* Face Culling is used to discard triangles facing away from the camera, reducing the total number of triangles
* to be processed. In normal circumstances the GL_BACK triangles are culled, but in some circumstances, i.e. inside
* a mesh such as the skysphere, the triangles facing the camera are the GL_BACK ones and the GL_FRONT triangles need
* to be culled instead. Use SetFacesToCull to set which triangles gets culled - by default GL_BACK.
```